.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1
+
+jobs:
+  "test":
+    parameters:
+      version:
+        type: string
+        default: "latest"
+      golint:
+        type: boolean
+        default: true
+      modules:
+        type: boolean
+        default: true
+      goproxy:
+        type: string
+        default: ""
+    docker:
+      - image: "circleci/golang:<< parameters.version >>"
+    working_directory: /go/src/github.com/solarwinds/grpchan
+    environment:
+      GOPROXY: "<< parameters.goproxy >>"
+    steps:
+      - checkout
+      - run:
+          name: "Print the Go version"
+          command: >
+            go version
+      # Only run gofmt, vet & lint against the latest Go version 

```

Comment:
Isn't this running lint against all of go versions decaled in matrix?

Reply:
removed the comment